src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ReleaseViewModels.cs

```diff
@@ -98,52 +112,52 @@ public class ReleaseCreateViewModel
             }
         }
 
-        [PartialDateValidator] public PartialDate NextReleaseDate { get; set; }
+        [PartialDateValidator] public PartialDate? NextReleaseDate { get; set; }
 
-        [RegularExpression(@"^([0-9]{4})?$")] public string ReleaseName { get; set; }
+        [RegularExpression(@"^([0-9]{4})?$")] public string? ReleaseName { get; init; }
 
         public string Slug => SlugFromTitle(Title);
 
-        private string Title => Format(Year, TimePeriodCoverage);
+        private string? Title => Year.HasValue ? Format(Year.Value, TimePeriodCoverage) : null;
 
-        private int Year => int.Parse(ReleaseName);
+        private int? Year => ReleaseName != null ? int.Parse(ReleaseName) : null;
```

Comment:
Why are these optional? 

I'm not going to ask this again because we seem to be making the same kind of things optional everywhere. To me, this seems incorrect as these things are not optional and should have been set in the frontend.

Same questions for `ReleaseUpdateViewModel` as well.

Reply:
See previous comment.